index.html

```diff
@@ -9,13 +9,24 @@
     <script src="vendors/jquery/jquery.min.js"></script>
     <script src="components/angular/angular.min.js"></script>
     <script src="components/angular/angular-route.min.js"></script>
-    <script src="vendors/materialize/js/materialize.min.js"></script>
-    <script type="text/javascript" src="resources/js/config.js"></script>
-    <script type="text/javascript" src="resources/js/snippets.js"></script>
     <script type="text/javascript" src="vendors/angular-sanitize/angular-sanitize.js"></script>
     <script type="text/javascript" src="vendors/showdown/showdown.js"></script>
     <script type="text/javascript" src="vendors/markdown/markdown.js"></script>
-    <script type="text/javascript" src="resources/js/app.js"></script>
+    <script src="vendors/materialize/js/materialize.min.js"></script>
+    <script type="text/javascript" src="resources/js/config.js"></script>
+    <script type="text/javascript" src="resources/js/snippets.js"></script>
+    <script type="text/javascript" src="resources/js/app.module.js"></script>
+    <script type="text/javascript" src="resources/js/routeprovider.js"></script>
+    <script type="text/javascript" src="resources/js/Controllers/SnippetsController.js"></script>
+    <script type="text/javascript" src="resources/js/Controllers/TabController.js"></script>
+    <script type="text/javascript" src="resources/js/Directives/HomeDirective.js"></script>
+    <script type="text/javascript" src="resources/js/Directives/LanguageDirective.js"></script>
+    <script type="text/javascript" src="resources/js/Directives/TryonlineDirective.js"></script>
+    <script type="text/javascript" src="resources/js/filters/format_descFilter.js"></script>
+    <script type="text/javascript" src="resources/js/filters/toArrayFilter.js"></script>
+    <script type="text/javascript" src="resources/js/filters/orderEmptyfilter.js"></script>
+    <script type="text/javascript" src="resources/js/Directives/getinvolvedDirective.js"></script>
```

Comment:
@gkrishnan724 Won't this decrease the site's performance because of a large number of files being loaded? I really think that it will affect the speed